app/assets/stylesheets/_misc_legacy.sass

```diff
@@ -334,40 +334,10 @@ div.issue hr
   margin-top: -6px
   margin-bottom: 0
 
-#content div.issue table
-  td
-    background: none
-    border: 0 none
-    padding: 0 3px
-    padding-left: 0
-  th
-    background: none
-    border: 0 none
-    padding: 0 3px
-    padding-left: 0
-    font-weight: bold
-
-.attachments h4
-  margin-bottom: 6px
-  background: image-url("files-showhide.png") no-repeat right bottom
-  cursor: pointer
-  &.closed
-    background-position: right 5px
-
 table.files
   display: table
 
-#content table.files
-  td
-    border: 0
-    background: none
-  th
-    border: 0
-    background: none
-    font-weight: bold
-    padding: 1px
-  border: 0
-  background: none
+#content table.filese
```

Comment:
@HDinger Do you know where this table occurs? (`table.files`)

I would have expected the attachments table, but that one has no class...